esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2278,6 +2278,55 @@ bool CFileSprayEx::onReplicate(IEspContext &context, IEspReplicate &req, IEspRep
     return true;
 }
 
+const char* CFileSprayEx::getDropZoneDirByIP(const char* ip, StringBuffer& dir)
+{
+    if (!ip || !*ip)
+        return NULL;
+
+    Owned<IEnvironmentFactory> factory = getEnvironmentFactory();
+    Owned<IConstEnvironment> env = factory->openEnvironment();
+    if (!env)
+        return NULL;
+
+    Owned<IPropertyTree> root = &env->getPTree();
+    VStringBuffer xPath("Hardware/Computer[@netAddress='%s']/@name", ip);
+    const char* computer = root->queryProp(xPath.str());
+    if (!computer || !*computer)
```

Comment:
Ok, well we could still say that has to find an entry, which it will do now for localhost variants.
But it's really a separate issue from the one being fixed here anyway.